src/functions-and-arrays.js

```diff
@@ -47,6 +171,25 @@ const wordsCount = [
   'matter'
 ];
 
+
+function howManyTimes (wordsCount, searchWord)  {
+  count = 0;
+  if(!wordsCount.length){
+    return 0
+  }
+  words.forEach(item => {
```

Comment:
Where is words coming from? Would it perhaps be better to submit the wordsArray as an argument?